mautic/mautic
# Added new filter option for global email do not contact

Adds a filter which gathers all of the leads specified in the "do-not-contact" table for email.

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? |  N
| New feature? | Y
| Automated tests included? | N
| Related user documentation PR URL | N 
| Related developer documentation PR URL | N 
| Issues addressed (#s or URLs) | #4977
| BC breaks? |  N
| Deprecations? |  N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description: Adds a filter which gathers all of the leads specified in the "do-not-contact" table for email.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1. Create new segement
2.  Add new "Unsubscribe - All" filter
3. Save Segement
4. run segment cron job
5. Segment should be populated with all email related do not contacts.

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
